resources/assets/js/components/modals/edit-songs-form.vue

```diff
@@ -57,9 +57,21 @@
                                 <label>Track</label>
                                 <input type="number" min="0" v-model="formData.track">
                             </div>
+                            <div class="form-row" style="display: none;">
+                                <label>Disc</label>
+                                <input type="number" min="0" v-model="formData.disc">
+                            </div>
+                            <div class="form-row" style="display: none;">
+                                <label>Year</label>
+                                <input type="number" min="0" v-model="formData.albumYear">
+                            </div>
+                            <div class="form-row" style="display: none;">
+                                <label>Genre</label>
+                                <input type="text" v-model="formData.genre">
+                            </div>
                         </div>
                         <div v-show="currentView === 'lyrics' && editSingle">
-                            <div class="form-row">
+                            <div class="form-row" style="display: none;">
```

Comment:
@yawboateng I believe there was a line note above where @phanan preferred to keep these new metadata fields out of the frontend UI. So these were kept in for API purposes, but hidden so the changes weren't available. This might be something that is taken out and made available if it ever gets the chance to be merged in again.